core/src/impl/Kokkos_Atomic_Generic.hpp

```diff
@@ -232,6 +263,7 @@ KOKKOS_INLINE_FUNCTION T atomic_oper_fetch(
   } oldval, assume, newval;
 
   oldval.t = *dest;
+  if (op.test(oldval.t, val)) return oldval.t;
```

Comment:
Please use braces for all if statements.

Reply:
You mean `if (op.test(oldval.t, val)) { return oldval.t; |`, right?

Reply:
Yes